app/src/test/java/org/simple/clinic/newentry/PatientEntryEffectHandlerTest.kt

```diff
@@ -100,6 +100,20 @@ class PatientEntryEffectHandlerTest {
     verifyZeroInteractions(validationActions)
   }
 
+  @Test
+  fun `when fetch colony or village names effect is received, then fetch them from the database`() {
+    // given
+    val colonyOrVillages = listOf("Colony1, Colony2, Colony3, Colony 4")
```

Comment:
I think you accidentally added the `,` in the String. Here it's a list of single string, it should be `listof("Colony1", "Colony2", "Colony3", "Colony4")`